docs(partner-nodes): document model-based concurrency limit (#1233)
Comfy-Org/docs: tutorials/partner-nodes/concurrency-limits.mdx

Every new account starts with a concurrency limit of **5 concurrent requests**. As you purchase credits, the platform automatically increases your limit based on the total amount of paid invoices from the last four weeks. Self-serve limits scale up to **40 concurrent requests**.

## Model-based limits

> Higher-cost models have a tighter limit so that a single account cannot tie up many expensive generations at once. When you call a model that costs more than **$5 per request**, your concurrency limit for that model is **one quarter** of your account limit. Requests to lower-cost models continue to use your full account limit, and the two limits are tracked independently.

Reviewer: @sundar-svg Do these round up to the nearest whole number?

Author: yes.. it rounds up

> Higher-cost models have a tighter limit so that a single account cannot tie up many expensive generations at once. When you call a model that costs more than **$5 per request**, your concurrency limit for that model is **one quarter** of your account limit. Requests to lower-cost models continue to use your full account limit, and the two limits are tracked independently.

Reviewer: ```suggestion
Higher-cost models have a lower allowed concurrency. When you call a model that costs more than **$5 per request**, your concurrency limit for that model is **one quarter** of your account limit. Requests to lower-cost models continue to use your full account limit, and the two limits are tracked independently.
```
